### DISCUSSION TITLE
Navigate to root view for iOS 15

### DISCUSSION THREAD

**Discussion by nagarjuna-techension** [SwiftfulThinking/SwiftfulRouting#70] [Q&A]
Here is the scenario,

Dashboard view -> Payment view -> Enter Amount -> Confirmation Page -> Success Page.

After success, i will need to navigate to Dashboard view(Root View).

Please suggest how to navigate to Root view. Thank you

**Comment by SwiftfulThinking** [comment]
.dismissScreenStack will pop to rootSent from my iPhoneOn Aug 20, 2024, at 12:14 AM, nagarjuna-techension ***@***.***> wrote:﻿
Here is the scenario,
Dashboard view -> Payment view -> Enter Amount -> Confirmation Page -> Success Page.
After success, i will need to navigate to Dashboard view(Root View).
Please suggest how to navigate to Root view. Thank you

**Reply by nagarjuna-techension** [reply]
I am using iOS 15 for minimum deployment target.

It is showing like this  - 'dismissScreenStack()' is only available in iOS 16 or newer

Thank you.

**Comment by SwiftfulThinking** [comment]
Then it is not supported by SwiftUI and you will have to build a custom solution.Sent from my iPhoneOn Aug 20, 2024, at 12:26 AM, nagarjuna-techension ***@***.***> wrote:﻿
I am using iOS 15 for minimum deployment target.
Thank you.

**Reply by nagarjuna-techension** [reply]
Thank you.
